q2templates/util.py

```diff
@@ -42,8 +42,13 @@ def df_to_html(df, border=0, classes=('table', 'table-striped', 'table-hover'),
     .. [2] https://github.com/pandas-dev/pandas/issues/1852
 
     """
-    with pd.option_context('display.max_colwidth', -1):
-        return df.to_html(border=border, classes=classes, **kwargs)
+    try:
```

Comment:
If we [only support pandas 1.0](https://github.com/qiime2/q2templates/issues/41#issuecomment-582037132) we can drop this branch of code - does that work for you?

Reply:
Yeah, I agree.  Seems like that would be preferable.